src/commonMain/kotlin/komem/litmus/LitmusResult.kt

```diff
@@ -1,7 +1,11 @@
+package komem.litmus
+
+typealias LitmusResult = List<LitmusOutcomeInfo>
+
 val LitmusResult.interestingFrequency: Double
```

Comment:
Is this function used in the project (IDEA says it's unused). 
Please either remove it, or generalize it to accept `LitmusOutcomeType` as a parameter (and rename correspondingly).